proxmox/config_qemu.go

```diff
@@ -595,6 +595,32 @@ func NewConfigQemuFromApi(vmr *VmRef, client *Client) (config *ConfigQemu, err e
 		diskConfMap["storage"] = storageName
 		diskConfMap["file"] = fileName
 
+		storageContent, err := client.GetStorageContent(vmr, storageName)
+		if err != nil {
+			log.Fatal(err)
+			return nil, err
+		}
+		var storageFormat string
+		contents := storageContent["data"].([]interface{})
+		for content := range contents {
+			storageContentMap := contents[content].(map[string]interface{})
+			if storageContentMap["volid"] == filePath {
```

Comment:
@Matchlighter **filePath** does not exist anywhere in the code

Reply:
@mleone87 I've included a fix for this in my PR #98

Reply:
Thanks @yukron. I must have built in the wrong dir/branch - I feel so foolish...